Sources/EllipticCurveKit/Encoding/DER.swift

```
@@ -35,6 +35,37 @@ public func derEncode(r: Number, s: Number) -> String {
     return DERCode.SEQUENCE.hex + byteCount.evenNumberedHexStringChars + encoded
 }
 
+public func derDecode(data: Data) -> (r: Number, s: Number)? {
```

If this function where to throw instead of returning an optional, the user can get info about what was wrong (about the input). 

Also might be worth introducing a namespace for `DER` now, let's go with:

```
/// DER decoding and encoding 
public enum DER {} // just a namespace

public extension DER {
    static func decode() throws -> (r: Number, s: Number) ...
    static func encode(r: Number, s: Number) -> String
}
```

---

I had the same thought when writing this but opted to make the change as minimal as possible and try to follow the existing conventions in the codebase.

---

Sure, I can make those changes, thanks!